cfgov/jinja2/v1/_includes/molecules/contact-email.html

```diff
@@ -0,0 +1,33 @@
+{% from 'macros/util/format/contact.html' import format_phone as format_phone %}
+
+{# ==========================================================================
+
+   contact_email.render()
+
+   ==========================================================================
+
+   Description:
+
+   Create a Contact-Email molecule.
+   See [GHE]/flapjack/Modules-V1/wiki/Contact-Email
+
+   emails: An array containing the email addresses.
+
+   ========================================================================== #}
+
+{% macro render(emails) %}
+<div class="m-contact-email">
+    {% set icon = 'cf-icon-email' %}
+    {% set title = 'Email' %}
+
```

Comment:
Nope! Good catch. These used to be in a list, but it's simpler not to have them that way, since, for example, the address isn't inside a list. Fixed in https://github.com/cfpb/cfgov-refresh/pull/1110